sigma.css

```diff
@@ -1,1371 +1,1363 @@
 @charset "utf-8";
 
 @import url('http://www.scp-wiki.net/local--files/component:theme/font-bauhaus.css');
-@import url('http://fonts.googleapis.com/earlyaccess/nanumgothic.css');
-@import url('http://fonts.googleapis.com/earlyaccess/nanumgothiccoding.css');
-@import url('http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css');
-@import url('http://scpnet.org/interwiki.css');
+@import url('http://fonts.googleapis.com/earlyaccess/notosansjp.css');
+@import url('http://fonts.googleapis.com/earlyaccess/notosanskr.css');
+@import url('https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css');
 
 /*
     SCP Sigma 9
     [2014 Wikidot Theme]
     Created for the SCP Foundation by Aelanna
     Edited for SCP Foundation by Dr Devan
-    Edited for SCP Foundation by The INT Tech Team

```

Comment:
For the moment, we'll keep the `by The INT Tech Team`. I will add a link to a CONTRIBUTORS file on the github.